purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit_purchase_order"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//div[@t-if='o.date_order']" position="before">
+            <div t-if="o.payment_term_id">
+                <strong>Payment Term:</strong>
+                <br />
+                <span t-field="o.payment_term_id.name"/>
+            </div>
+        </xpath>
+        <xpath expr="//div[@t-field='o.partner_id']" position="replace">
+            <div t-field="o.partner_id"
+            t-options='{"widget": "contact", "fields": ["purchase_address", "name", "phone"], "no_marker": True, "phone_icons": True}'/>
+        </xpath>
```

Comment:
What are you trying to achieve with this? Maybe you are confused between the company address and the partner (supplier) address?

Reply:
I was thinking that you could simply use this module https://github.dev/qrtl/rpl-custom/blob/e56c56f8d98bb66c6209cc374719c1c8f2c43b38/report_company_address to show the company address in English, depending on the partner's language.

Reply:
Maybe I'd misunderstanding. Thank you for your help.